packages/superset-ui-encodable/src/parsers/scale/createScaleFromScaleType.ts

```diff
@@ -44,11 +44,11 @@ export default function createScaleFromScaleType<Output extends Value>(
     case ScaleType.THRESHOLD:
       return scaleThreshold<number | string | Date, Output>();
     case ScaleType.ORDINAL:
-      return scaleOrdinal<HasToString, Output>();
+      return scaleOrdinal<HasToString | null | undefined, Output>();
```

Comment:
any reason not to use the `type CategoricalScaleInput = HasToString | null | undefined` from `/types/Scale`?